templates/includes/home.html

```diff
@@ -26,16 +26,27 @@
 <div class="home wrapper">
     <h2 class="post-list-heading">{{ section.title | default(value="Latest News") }}</h2>
     
-    {% set posts = get_section(path="posts/_index.md") %}
+    {% set posts = get_section(path="news/_index.md") %}
+    {% set pages = posts.pages | slice(end=config.extra.home.max_posts) %}
+    {{ macros::post_list(posts=pages, placeholder="None yet - stay tuned for updates and announcements!") }}
+
+    <h2 class="post-list-heading">{{ section.title | default(value="Latest Blog Posts") }}</h2>
+
+    {% set posts = get_section(path="blog/_index.md") %}
     {% set pages = posts.pages | slice(end=config.extra.home.max_posts) %}
     {{ macros::post_list(posts=pages, placeholder="None yet - stay tuned for updates and announcements!") }}
 
     <p class="feed-subscribe">
         {% if pages | length > 0 %}
```

Comment:
This 'pages' variable refers to the blog pages, not the news pages.

I think maybe let's just make these buttons display unconditionally? We're never going to go back to zero posts again :p

Reply:
Fixed